jax/lax/__init__.py

```diff
@@ -342,6 +342,8 @@
   all_to_all_p as all_to_all_p,
   axis_index as axis_index,
   axis_index_p as axis_index_p,
+  pcopy as pcopy,
+  pcopy_p as pcopy_p,
```

Comment:
Do we need to export `pcopy_p`, or can we avoid that?

Reply:
Why not export it? All of the other ops export it.

Reply:
The reason not to export it is just the usual reason we don't export every symbol: primitives are private and internal, and we don't export those into public namespaces so as to indicate that their use isn't supported the way public APIs are supported (deprecation cycles etc).

Unfortunately by the time we decided to start hiding our internals, there were a lot of Google-internal users of those internals. So we had to export some symbols to keep those users working. We've slowly been burning down those users, and removing the exports. But for new primitives we try to make them as private as possible.

WDYT?

Reply:
I actually have systems that generally need to have the primitivies be publically accessible, (i.e, using the primitives as the keys for callback lookup-tables like you see in `eval_jaxpr`) Is this something that JAX generally does not support third-party libraries from doing anymore?

Reply:
Resolved. This will be accessible in `jax.extend.primitives` instead.